metricbeat/module/system/_meta/config.full.yml

```diff
@@ -4,10 +4,10 @@
     - cpu
 
     # System Load stats
-    - load
+    #- load
```

Comment:
I think we can leave this enabled. We have code to disable it automatically on WIndows: https://github.com/elastic/beats/blob/master/metricbeat/Makefile#L61